src/js/actions/libraries.js

```diff
@@ -937,8 +937,8 @@ define(function (require, exports) {
         }
 
         return Promise.fromCallback(function (done) {
-                representation.getContentPath(done);
-            })
+            representation.getContentPath(done);
+        })
             .bind(this)
```

Comment:
Yeah this happens in multiple places, I think the format @shaoshing suggested above is the cleanest way.